packages/jest-gas-reporter/src/index.ts

```diff
@@ -164,49 +168,64 @@ export class GasReporter implements jest.Reporter {
-    console.log();
-    console.log('Function Calls:');
-    const methodTable = new easyTable();
     for (const contractName of Object.keys(contractCalls)) {
+      const contractStats: ContractStats = {
+        deployment: 0,
+        methods: {}
+      };
+      if (contractCalls[contractName].deploy) {
+        const deployGas = contractCalls[contractName].deploy.gasData;
+
+        if (deployGas[1] && deployGas[1] !== deployGas[0]) {
+          throw new Error('Multiple deployments with differing gas costs detected!');
+          // This shouldn't happen with our current workflow where contracts are deployed exactly once before tests are run.
+        }
+        contractStats.deployment = deployGas[0];
+      }
       const methodCalls = contractCalls[contractName].methodCalls;
       Object.keys(methodCalls).forEach(methodName => {
         const method = methodCalls[methodName];
         const total = method.gasData.reduce((acc, datum) => acc + datum, 0);
         const average = Math.round(total / method.gasData.length);
         const min = Math.min(...method.gasData);
         const max = Math.max(...method.gasData);
-        methodTable.cell('Contract Name', contractName);
-        methodTable.cell('Method Name', methodName);
-        methodTable.cell('Calls', method.calls);
-        methodTable.cell('Min Gas', min);
-        methodTable.cell('Max Gas', max);
-        methodTable.cell('Average Gas', average);
-        methodTable.newRow();
+
+        const stats: Stats = {calls: method.calls, min: min, max: max, avg: average};
+        contractStats.methods[methodName] = stats;
       });
-    }
-    console.log(methodTable.toString());
-    console.log('Deployments:');
-    const deployTable = new easyTable();
-    for (const contractName of Object.keys(contractCalls)) {
-      if (contractCalls[contractName].deploy) {
-        const deploy = contractCalls[contractName].deploy;
-
-        const total = deploy.gasData.reduce((acc, datum) => acc + datum, 0);
-        const average = Math.round(total / deploy.gasData.length);
-        const min = Math.min(...deploy.gasData);
-        const max = Math.max(...deploy.gasData);
-
-        deployTable.cell('Contract Name', contractName);
-        deployTable.cell('Deployments', deploy.calls);
-        deployTable.cell('Min Gas', min);
-        deployTable.cell('Max Gas', max);
-        deployTable.cell('Average Gas', average);
-        deployTable.newRow();
+
+      if (contractStats.deployment > 0) {
+        gasConsumed[contractName] = contractStats;
       }
     }
-    console.log(deployTable.toString());
+
+    console.log(this.objectToTable(gasConsumed).toString());
+  }
+
+  objectToTable(gasConsumed: GasConsumed): easyTable {
+    const table = new easyTable();
+    Object.keys(gasConsumed).forEach(contract => {
+      const contractStats = gasConsumed[contract];
+      table.cell('Contract', contract);
+      table.cell('Deployment', contractStats.deployment);
```

Comment:
Why is this value relevant? Also what is it exactly?

Reply:
Gas consumed when deploying the contract.

Reply:
Ohhh okay.